index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/0bd1156d30.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <section class="Main-Page">
+        <header>
+     <nav class="nav">
+ <a  href="#" class="logo">John Doe</a>
+<nav class="nav-links-menu"><p class="side-logo"><i class="fa-solid fa-bars"></i></p></nav>
+     </nav>
+    </header>
+     <section class="section">
+        <img src="background.png" alt="" >
+</section>
+    <!-- <section class="my-works">
+        <h1 class="my-works-header">My Projects</h1>
+        <hr>
+        <div class="show-case">
+    <div class="p1"></div>
+    <div class="p2"></div>
+    <div class="p3"></div>
+    <div class="p4"></div>
+    <div class="p5"></div>
+    <div class="p6"></div>
+    <div class="p7"></div>
+    
+    </div>
+    
+        </section> -->
+    
```

Comment:
- [ ] Please remove the commented code from your project as requested by the previous reviewer